client/checkout/classic/upe-deferred-intent-creation/payment-processing.js

```diff
@@ -184,6 +189,49 @@ function appendSetupIntentToForm( form, confirmedIntent ) {
 	form.append( input );
 }
 
+function maybeEnableStripeLink( api, paymentMethodType ) {
```

Comment:
> Maybe we should then re-name payment-processing.js so that it reflects better what its responsibilities?

The definition of `maybeEnableStripeLink` has to stay here; you could argue it's still pretty relevant to payment processing. I've moved it's declaration to `event-handlers.js`, where it is actually initialised. See 2ee6b6ebea00cc109bfa8c8c52069e2c2eb633a3.